### ISSUE TITLE
Look into using templateflow for templates

### ISSUE THREAD

**Issue by arokem** [yeatmanlab/pyAFQ#202]
https://github.com/templateflow

I think this does something like: 

```
from templateflow import api as tfapi
tfapi.get('MNI152NLin6Asym', atlas=None, resolution=[1, 2], extension=['.nii', '.nii.gz'])
tfapi.get('MNI152NLin2009cAsym', atlas=None, extension=['.nii', '.nii.gz'])
tfapi.get('OASIS30ANTs', extension=['.nii', '.nii.gz'])
```

But need to explore more. There are apparently also infant templates: https://github.com/templateflow/tpl-MNIInfant

**Comment by arokem** [comment]
More docs here: https://templateflow.github.io/python-client/

**Comment by arokem** [comment]
This was resolved through #224
